src/tests/unit/tests/injected/components/details-dialog.test.tsx

```diff
@@ -225,13 +245,11 @@ describe('DetailsDialog', () => {
                     dialogHandler: dialogDetailsHandlerMockObject,
                 };
 
-                const wrapper = shallow<DetailsDialog>(<DetailsDialog {...props} />);
-
-                const commandBar = wrapper.find(CommandBar);
+                render(<DetailsDialog {...props} />);
 
-                commandBar.prop('shouldShowInspectButtonMessage')();
+                getMockComponentClassPropsForCall(CommandBar).shouldShowInspectButtonMessage();
 
-                handlerMock.verify(handler => handler(wrapper.instance()), Times.once());
+                handlerMock.verify(handler => handler(It.isAny()), Times.once());
```

Comment:
See above comment for how to adapt this test to still check that the component is passed into the click handler.